src/init.cpp

```diff
@@ -111,6 +112,8 @@ using node::CleanupBlockRevFiles;
 using node::DEFAULT_PRINTPRIORITY;
 using node::DEFAULT_STOPAFTERBLOCKIMPORT;
 using node::LoadChainstate;
+using node::MempoolPath;
+using node::ShouldPersistMempool;
```

Comment:
04f72e90204c049d5d320388fda1f01e53417938: What is the point of those, when it is less code overall with them removed?

Reply:
Do you mean why have these `using` declarations instead of using `node::MempoolPath`, `node::ShouldPersistMempool` directly?

Reply:
Yes. If only one symbol requires them, it seems less code to just write the namespace out.

Reply:
In the longer term I think `init.cpp` will go in the `node` namespace so it's probably less code change in the longer term to have the `using` declarations. In any case, probably not going to address it in this PR.